.github/actions/setup-project/action.yml

```diff
@@ -39,9 +39,9 @@ runs:
 
     - name: 📚 Start MongoDB
       if: ${{ inputs.slim == 'false' }}
-      uses: supercharge/mongodb-github-action@v1.9.0
+      uses: supercharge/mongodb-github-action@v1.11.0
       with:
-        mongodb-version: 4.2.8
+        mongodb-version: 5.0.29
```

Comment:
This MongoDB version bump to >5 was needed to solve [this runtime error](https://github.com/novuhq/novu/actions/runs/11514455656/job/32053213154?pr=6767#step:9:1644) that disallows setting empty objects:
> `Invalid $addFields :: caused by :: an empty object is not a valid value. Found empty object at path preferences.all`

Version 5 supports empty object setting per [this thread](https://www.mongodb.com/community/forums/t/mongoservererror-invalid-set-caused-by-an-empty-object-is-not-a-valid-value/148344/4). 

`5.0.29` is the latest 5.x release per the MongoDB [changelog](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.29---sep-30--2024).

Reply:
Noted, Thanks 👍🏼. We will post a note a to community/

Reply:
Good call out!